package.json

```diff
@@ -45,11 +45,11 @@
   },
   "dependencies": {
     "tooltip.js": "^1.3.3",
-    "vl-ui-core": "^5.1.2"
+    "vl-ui-core": "milieuinfo/webcomponent-vl-ui-core#parent"
```

Comment:
hier maak je dan een versie nummer van eens je gaat releasen?

Reply:
Zodra de PR goedgekeurd is, vervang ik inderdaad de branch dependency.